src/main/java/org/csanchez/jenkins/plugins/kubernetes/pod/yaml/YamlMergeStrategy.java

```diff
@@ -0,0 +1,17 @@
+package org.csanchez.jenkins.plugins.kubernetes.pod.yaml;
+
+import hudson.ExtensionPoint;
+import hudson.model.AbstractDescribableImpl;
+import io.fabric8.kubernetes.api.model.Pod;
+
+import javax.annotation.Nonnull;
+import java.io.Serializable;
+import java.util.List;
+
+public abstract class YamlMergeStrategy extends AbstractDescribableImpl<YamlMergeStrategy> implements ExtensionPoint, Serializable {
```

Comment:
So, why does this need to be `Serializable`? I do not see any evidence that it would either become part of `program.dat` or be transferred to an agent. (XStream does not require any particular interface, which is why it is uncommon for `Describable`s to be `Serializable`—you would only do that if they were _also_ serialized using something other than XStream.)

Reply:
A `PodTemplate` reference is passed to a callback at https://github.com/jenkinsci/kubernetes-plugin/blob/038a4d4d475bcb4dd754aeb6e350b4e5bd48be27/src/main/java/org/csanchez/jenkins/plugins/kubernetes/pipeline/PodTemplateStepExecution.java#L187

Reply:
OK.